app/models/services/cbd_importer.rb

```diff
@@ -1,21 +1,22 @@
 class Services::CbdImporter
 
   def call
+    manager_hash = Hash[Manager.pluck(:name, :id).collect {|name, id| [name, id]}]
```

Comment:
Looks like the `collect` method here is returning the data in exactly the same format as `.pluck` has already returned it